packages/variant-theme/package.json

```diff
@@ -2,13 +2,17 @@
   "name": "@fluentui-contrib/variant-theme",
   "version": "0.0.1",
   "dependencies": {
-    "@swc/helpers": "~0.5.2"
+    "@swc/helpers": "~0.5.2",
+    "@fluentui/react-components": ">=9.46.3 <10.0.0",
+    "@fluentui/react-migration-v8-v9": "^9.6.12",
+    "@fluentui/scheme-utilities": "^8.3.46",
```

Comment:
what's the reasoning behind those 2 deps ? 

`react-migration-v8-v9` is also very dangerous as it has react-components as dependency ( that's bug on fluent core side IMO ) - lets remove these

Reply:
I used VariantThemeType from scheme-utilities and some color records from react-migration-v8-v9

Reply:
right so question regarding v8-v9. I thought this package is intent is to create v8 feature parity to v9 ecosystem. is that the case or I am missing something ? ty

Reply:
I would like to use @fluentui/tokens but I found that colors are not exported from @fluentui/tokens. There is some copied files of colors in v8-v9 packages so I import these colors from v8-v9 package

Reply:
hmm this doesn't feel right. can you please open issue on https://github.com/microsoft/fluentui/issues so we  can avoid this in future ?